sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FunctionComplexity.CSharp9.cs

```diff
@@ -0,0 +1,42 @@
+﻿//FN
```

Comment:
given we have boolean expressions in the main test, could you add some test cases with the new Pattern matching improvements ?